core/object/script_language_extension.h

```diff
@@ -737,10 +737,15 @@ class ScriptInstanceExtension : public ScriptInstance {
 	}
 
 	virtual void notification(int p_notification) override {
-		if (native_info->notification_func) {
-			native_info->notification_func(instance, p_notification);
+		notification(p_notification, false);
+	}
+
+	virtual void notification(int p_notification, bool p_reversed = false) override {
+		if (native_info->notification_func2) {
+			native_info->notification_func2(instance, p_notification, p_reversed);
```

Comment:
Again, this'll need to still call `notification_func` if `notification_func2` isn't set, so that older GDExtensions will keep working:

```suggestion
		if (native_info->notification_func2) {
			native_info->notification_func2(instance, p_notification, p_reversed);
		} else if (native_info->notification_func) {
			native_info->notification_func(instance, p_notification);
```